test/lib/ufe/testGroupCmd.py

```diff
@@ -127,3 +134,18 @@ def testUsdGroup(self):
         self.assertTrue(newGroupPath in childPathsRedo)
         self.assertTrue(ball5Path not in childPathsRedo)
         self.assertTrue(ball3Path not in childPathsRedo)
+
+    @unittest.skipUnless(mayaUtils.previewReleaseVersion() >= 122, 'Requires Maya fixes only available in Maya Preview Release 122 or later.')
+    def testGroupAcrossProxies(self):
+        sphereFile = mayaUtils.getTestScene("groupCmd", "sphere.usda")
+        sphereDagPath, sphereStage = mayaUtils.createProxyFromFile(sphereFile)
+        usdSphere = sphereDagPath + ",/pSphere1"
+
+        torusFile = mayaUtils.getTestScene("groupCmd", "torus.usda")
+        torusDagPath, torusStage = mayaUtils.createProxyFromFile(torusFile)
+        usdTorus = torusDagPath + ",/pTorus1"
+
+        try:
+            cmds.group(usdSphere, usdTorus)
+        except Exception as e:
+            self.assertTrue('cannot group across usd proxies')
```

Comment:
Creating two different proxies from files (sphere in one, torus in the other), then selecting an object from each and trying to group them.  This was previously a crash, but will be fixed in the next Maya PR.